pkg/local-resource-manager/controller_manager.go

```diff
@@ -38,6 +40,108 @@ import (
 // resources calculation.
 
 // Start starts the controller.
+
+func canMergeFlavours(flavour1, flavour2 *nodecorev1alpha1.Flavour) bool {
```

Comment:
Also, I'm not understanding why we have those constraints. Can't we merge two flavors of the same type if those have different characteristics (like CPU, RAM, etc)? Why so?

Reply:
In this first approach the main focus was to just aggregate nodes with the same specifications because currently, the information stored in flavours is the same stored in nodes.

Reply:
Yes I understood your point.

Does this mean that if a provider has nodes with different characteristics (like CPU and RAM) it cannot aggregate the corresponding flavors?

If so, what's the meaning of the aggregation eventually?

Reply:
Yes, at the moment.

The meaning of the flavour aggregation is to have an abstraction of all the available resources to be consumed. This will allow us to have a cleaner list of the resouerces and also a much tighter control of what is characteristics are published.